LocalExtensions.php

```diff
@@ -0,0 +1,3 @@
+<?php
+// Switch case
+switch( $wgDBname ) 
```

Comment:
Needs an opening { and } after the switch? Possibly add case meta wiki; just so you know how it's set up. (Look at the switch on Orain)